lib/v2/picuki/profile.js

```diff
@@ -31,12 +36,8 @@ module.exports = async (ctx) => {
 
     async function getMedia(url) {
         const getPost = async () => {
-            const response = await got(url, {
-                headers: {
-                    Referer: profileUrl,
-                },
-            });
-            return response.data;
+            const response = await puppeteerGet(url, browser);
+            return response;
```

Comment:
Please close the browser after all puppeteer page requests finish.
```js
await browser.close();
```

Reply:
Thank you for the reminder.